android/platform/android/res/values/colors.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="notification_color">#EEEEEE</color>
```

Comment:
Could you make this setting less generic? For example prefix it with `parse_` to prevent possible clashes with app implementation.